lib/emque/producing/publisher/rabbitmq.rb

```diff
@@ -42,9 +39,11 @@ def publish(topic, message_type, message, key = nil)
               :persistent => true,
               :type => message_type,
               :app_id => Emque::Producing.configuration.app_name,
-              :content_type => "application/json")
+              :content_type => "application/json"
+            )
+
+            success = ch.wait_for_confirms if raise_on_failure
```

Comment:
won't success be nil if raise_on_failure is false? I'm not sure that's the intention. I was thinking:

```
if raise_on_failure
  success = ch.wait_for_confirms
  unless success
    Emque::Producing.logger.warn("RabbitMQ Publisher: message was nacked")
    ...
  end
end
```